sys/net/gnrc/link_layer/lwmac/lwmac.c

```diff
@@ -878,7 +870,17 @@ static void _lwmac_msg_handler(gnrc_netif_t *netif, msg_t *msg)
             lwmac_schedule_update(netif);
             break;
         }
-
+#if (GNRC_MAC_ENABLE_DUTYCYCLE_RECORD == 1)
+        case GNRC_MAC_TYPE_GET_DUTYCYCLE: {
+            /* Output LWMAC's radio duty-cycle ratio */
+            uint64_t duty;
+            duty = (uint64_t) rtt_get_counter();
+            duty = ((uint64_t) netif->mac.prot.lwmac.awake_duration_sum_ticks) * 100 /
+                   (duty - (uint64_t)netif->mac.prot.lwmac.system_start_time_ticks);
+            printf("[LWMAC]: achieved radio duty-cycle: %lu %% \n", (uint32_t)duty);
+            break;
+        }
+#endif
```

Comment:
I would ACK this one but I see that you use `{` after the case statements, which is not on our standard coding conventions. I guess this might be in a huge part of the code in LwMac, but, can you just remove them from this file? Maybe you can check later if this is also present in more parts in the code and fix them in follow-up PRs. Maybe uncrustify helps but I'm not sure.

Reply:
OK, paco, I will remove the `{` I introduced in this PR. There are indeed many similar cased I have done so in LWMAC and GoMacH files, and I will open following PRs to remove those `{` , as you suggested. Thanks.

Reply:
@kYc0o if @zhuoshuguo removes those it wouldn't compile anymore, since you can't set a jump mark (which is what a case statement is) to a variable allocation (`uint64_t duty`). This is why he needs to open a new block.

Reply:
Ohhh I see, that's why they're there. OK with me then.